sdk/storage/storage-blob/execute-samples.js

```diff
@@ -32,22 +32,26 @@ async function exec(cmd, cwd) {
 
 async function runSamples(language) {
   let cmd;
+  let extension;
   // Tries to execute all the samples in the `directory`.
   const directory = `./samples/${language}`;
 
   if (language === "typescript") {
     cmd = "ts-node";
+    extension = ".ts";
+    await exec(`npm run build:ts-samples`, "./");
   } else {
     cmd = "node";
-    await exec(`npm run build:js-samples`, directory);
```

Comment:
I don't know how the old code would work in CI since there's no package.json under the `samples/javascript` or `samples/typescript` folder. It didn't work on my local dev machine

Reply:
maybe it depends on the folder from where you run the script?

Reply:
`build:js-samples`, `execute:samples` steps are part of the sdk's package.json.. i.e., `sdk/storage/storage-blob/package.json`.

Just execute the file - `node execute-samples.js` to run the samples. 
Samples import src code currently and hence no package.json in the samples folder.

Reply:
If we are importing the published package instead of the src code, we need to update execute-samples.js accordingly.
[Just checked.. blob samples import src code whereas file-share samples import published package.]

Reply:
> Just execute the file - `node execute-samples.js` to run the samples.

I was using `rushx execute:samples` under storage-file-share folder and got an error about not finding NPM script build:ts-samples.

>  [Just checked.. blob samples import src code whereas file-share samples import published package.]

Right, so far Will only did the work for file-share. He added common/scripts/prep-samples.js to patch TS samples back to import from "../../src". We need to add support to prepare JS samples as well. #6362

Reply:
If we are importing published package in the samples.. 
For the ci, instead of prep-samples script, we can update this execute-samples.js script to install tar zip of the package in the samples folder and run the samples.
(Also needs a package.json in the samples folder)